report-schema-v100.json

```diff
@@ -0,0 +1,600 @@
+                                            ]
+                                        }
+                                    },
+                                    "remediations": {
+                                        "$id": "#/properties/detail/properties/remediations",
+                                        "type": "array",
+                                        "title": "The remediations schema",
+                                        "description": "A list of remediations, describes what needs to be done to fix the problem.",
+                                        "default": [],
+                                        "examples": [
+                                            [
+                                                {
+                                                    "type": "hint",
+                                                    "context": "Please update your scripts to be compatible with the changes."
+                                                }
+                                            ]
+                                        ],
+                                        "additionalItems": true,
+                                        "items": {
+                                            "$id": "#/properties/detail/properties/remediations/items",
+                                            "anyOf": [
+                                                {
+                                                    "$id": "#/properties/detail/properties/remediations/items/anyOf/0",
+                                                    "type": "object",
+                                                    "title": "The first remediation",
+                                                    "description": "",
+                                                    "default": {},
+                                                    "examples": [
+                                                        {
+                                                            "type": "hint",
+                                                            "context": "Please update your scripts to be compatible with the changes."
+                                                        }
+                                                    ],
+                                                    "required": [
+                                                        "type",
+                                                        "context"
+                                                    ],
+                                                    "properties": {
+                                                        "anyOf": [
+                                                            {
+                                                                "type": {
+                                                                    "$id": "#/properties/detail/properties/remediations/items/anyOf/0/properties/type",
```

Comment:
and this is probably not possible as well. properties should be always key-value dicts as it looks like.